### ISSUE TITLE
How to use on Windows 10?

### ISSUE THREAD

**Issue by Thundros** [njohnst/melted-rtc#1]
Hello! I really like this idea of being able to not have to use a STUN / TURN server for WebRTC. I hate that most of them are connected to google. I don't trust google at all. I, however, have a few questions. How can I get this to work for Windows 10? I started the example server correctly, however, when I try to start the client, I receive the following error :: 

"Error: SimplePeer.WEBRTC_SUPPORT evaluated to false"

How do I fix this?

Also, how do I get this to work say with a WebRTC app I would like to use?

Thank you kindly for the AWESOME idea & help!

**Comment by Thundros** [comment]
Hello?

**Comment by Thundros** [comment]
........

**Comment by njohnst** [comment]
Hi, sorry for the delay.

The client code won't run in node - you need to run it from a browser.  Under examples/basic there is a webpack config to build 'bundled-client.js' and there is an html file called 'test.js' under public that uses the bundled client file.

If you're able to get it to run you might notice that the RTC ping actually has a larger delay that WS (4ms vs 1ms) - this is because there is more processing overhead.

Some other caveats - the current implementation of RTC used on the server side only enables 1 socket per port - as a result the library uses a port range to support multiple users (like 22001 - 22010 would support 10 slots).  This might have issues in terms of using cloud services such as AWS down the road.

At the moment I'm actually investigating another library called [Colyseus](https://github.com/gamestdio/colyseus) (which doesn't support RTC) and considering whether to implement RTC for that library instead, so I would consider this library as WIP and could potentially be deprecated, but could be a good starting point.

**Comment by Thundros** [comment]
Colyseus doesn't do hole punching :(

**Comment by njohnst** [comment]
Yes, Colyseus only handles WebSockets (TCP) - which wouldn't need punching, but suffers from the latency issues that I was trying to solve with this library.  That said, this library only has basic features in comparison to Colyseus, which has basically everything that I want except for UDP support.  

So I'm considering taking some of the work I did on this library and creating a PR for Colyseus, as it would likely be a more efficient use of development time for me.

TL;DR:
That said, if UDP and hole-punching are critical for your use case, this library is currently one of the only ones AFAIK, but depending on what other server features you need, you might have a lot of work ahead of you in implementing or integrating them (such as authentication, authorization, horizontal scaling).

Let me know if you have any other issues or questions.
